Alpaca.Markets/Interfaces/IStreamAgg.cs

```diff
@@ -10,12 +10,7 @@ public interface IStreamAgg : IAggBase
         /// <summary>
         /// Gets asset name.
         /// </summary>
-        String Symbol { get; }
-
-        /// <summary>
-        /// Gets asset's exchange identifier.
-        /// </summary>
-        Int64 Exchange { get; }
```

Comment:
Mark `Exchange` property with `Obsolete` attribute with `This property will be removed in upcoming major SDK release.`